CPAC/resources/cpac_outputs.tsv

```diff
@@ -100,6 +100,7 @@ space-template_desc-bold_mask	mask	template	func	NIfTI
 space-template_res-derivative_desc-bold_mask	mask	template	func	NIfTI					
 motion	motion		func	TSV					
 desc-summary_motion	motion		func	TSV					
+dvars	motion		func	TSV				Yes
```

Comment:
Mostly looks good, but before merging this in, we should make sure all the rows have the same number of columns

<img width="1003" alt="We can make this file beautiful and searchable if this error is corrected: It looks like row 103 should actually have 10 columns, instead of 9. in line 102." src="https://github.com/FCP-INDI/C-PAC/assets/5974438/eed13baa-a7ea-49c8-bc8c-e5182c867220">


```suggestion
desc-summary_motion	motion		func	TSV						
dvars	motion		func	TSV				Yes
```

Reply:
Got it, thanks

Reply:
of course, not a problem.